### PR TITLE
Add slug and locale to orderedcontentset

### PR DESCRIPTION
**Pull request by HawkiesZA**
## Purpose
Ordered Content Sets are currently only identified by names, which are supposed to be translatable not supposed to be unique. This means that there could be clashes, particularly on imports, causing the wrong the ordered set to update

## Solution
This adds a Slug and Locale field to OrderedContentSets, similar to Pages and Assessments. Ordered Sets will then be unique by Slug and Locale.

#### Checklist
- [x] Added or updated unit tests
- [x] Added to release notes
- [ ] Updated readme/documentation (if necessary)

### CODE REVIEW

File: home/import_ordered_content_sets.py [hunk 1]
```diff
@@ -40,20 +40,25 @@ def __init__(
...
         Get or initialize an instance of OrderedContentSet from a row of a CSV file.
 
         :param row: The row of the CSV file, as a dict.
-        :param set_name: The name of the ordered content set.
+        :param set_slug: The slug of the ordered content set.
+        :param set_locale: The locale of the ordered content set.
         :return: An instance of OrderedContentSet.
         """
-        ordered_set = OrderedContentSet.objects.filter(name=set_name).first()
+        locale = self.locale_helper.locale_from_display_name(set_locale)
```
[the remark this answers is not in the document]
Comment (jerith) [reply home/import_ordered_content_sets.py:56]: Actually, it looks like we're exporting the code already. So we definitely don't want to import by display name.

File: home/import_ordered_content_sets.py [hunk 2]
```diff
@@ -155,7 +160,10 @@ def _create_ordered_set_from_row(
         :return: An instance of OrderedContentSet.
         :raises ImportException: If time, units, before_or_afters, page_slugs and contact_fields are not all equal length.
         """
-        ordered_set = self._get_or_init_ordered_content_set(row, row["Name"])
+        ordered_set = self._get_or_init_ordered_content_set(
+            row, row["Slug"], row["Locale"]
```
Comment (jerith) [inline home/import_ordered_content_sets.py:164]: Do we care about case here? Should we perhaps normalise to lowercase when we read from CSV/XLSX?

File: home/tests/test_api.py [hunk 3]
```diff
@@ -1588,7 +1634,10 @@ def test_orderedcontent_moderation(self):
             content_type_id=content_type.id, workflow_id=workflow.id
         )
 
-        ordered_content_set_instance = OrderedContentSet()
+        site = Site.objects.get(is_default_site=True)
+        ordered_content_set_instance = OrderedContentSet(
+            slug="ordered-set-2", locale=site.root_page.locale
+        )
```
[the remark this answers is not in the document]
Comment (jerith) [reply home/tests/test_api.py:1640]: We also need tests for filtering by slug and/or locale, I think.

File: home/import_helpers.py [hunk 4]
```diff
@@ -38,6 +39,40 @@ def __init__(
         super().__init__()
 
 
+class LocaleHelper:
```
[the remark this answers is not in the document]
Comment (HawkiesZA) [reply home/import_helpers.py]: Nope. Removed.

**Review by rudigiesler — APPROVED** [review]

**Pull request merged**